stopwords.go

```diff
@@ -62,6 +62,50 @@ func NewStopwords(path string) StopWords {
 }
 */
 
+// NewStopwords returns an instance of a stop words detector
+// new stopword lists can be added to the "resources/stopwords" directory as .txt with the filename
+// prefixed as "ISOLangCode_language_stopwords.txt"
+// ie. en_english_stopwords.txt
+func NewStopwords() StopWords {
+	cachedStopWords := make(map[string]*set.Set)
+	path := "resources/stopwords"
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		log.Println(err.Error())
```

Comment:
there's currently really no mechanism to catch this error, and you don't really know you've ended up with an empty list.  a couple options are either:

1. `NewStopwords` returns both `StopWords` and an error, and that error is checked in the caller.
2. This function is only called on `init`, panics on an error, and sets a global stopwords variable.

Reply:
I opted for the option number 1: The only real caller of the `NewStopwords()` is the `GetDefaultConfiguration()` in `configuration.go`.
When there is an error, it will just add a default stopwords made from the variable `sw` in `stopwords.go`.

Reply:
Hm, so I'm reviewing the changes for this part here, which duplicates some of the default stopwords lists created the global `sw` variable.  Since you're relying on a package variable anyway, I'd rather see the variable created from your new stopwords files and panic if there's a problem reading them.  (Which if there is, it definitely warrants a panic.)

Ultimately, the method of having duplicate stopwords lists, with divergent entries and not having the ultimate caller (`NewExtractor`, `New Cleaner`, etc.) know which one they're getting, is problematic.

Does option 2 seem doable?

Reply:
yes, I made the changes to add an `init()` which panics on an error, and sets a global stopwords variable.

Reply:
ty!